<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Splitted Sets</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>mole.png</normaloff>mole.png</iconset>
  </property>
  <widget class="QLabel" name="splitWhereLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Split by</string>
   </property>
  </widget>
  <widget class="QLabel" name="splitFeatureLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>feature =N/A</string>
   </property>
  </widget>
  <widget class="QLabel" name="whereLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>where</string>
   </property>
  </widget>
  <widget class="QListWidget" name="splittedSetsListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
